projects/cashmere/src/lib/stepper/stepper.component.ts

```diff
@@ -80,38 +83,39 @@ export class StepperComponent implements AfterContentInit {
         if (!this.steps) {
             return;
         }
-        if (value < 0 || value >= this.steps.length) {
-            throw Error('The hc-stepper activeIndex value of ' + value + ' is out of bounds');
-        }
-        if (this._routerEnabled) {
+        // if (value !== undefined && (value < 0 || value >= this.steps.length)) {
```

Comment:
Can we get this check back in?  There is one unit test failing because this isn't included.  Was there a reason to remove it?

Reply:
One of the reasons for these changes was to allow the activeIndex to be "null".  I am using -1 as the vehicle to not set an activeIndex. I did go down the path of using the type of `number | undefined` to manage the "null" feature. Should I switch over to that option and restore this `out of bounds` error?

Reply:
Ah okay, I thought it might be something like that.  Yes, if we could go that route and restore the error, I think that just makes life a little easier for our users to debug.  And just make sure the unit test is passing again (you can adjust the test if needed)

Reply:
Error is now restored.